proposals/2021-02-enable-ibc-transfers/README.md

```diff
@@ -0,0 +1,12 @@
+# Enable IBC Transfers
+
+The Proposal enables transfering and recieving assets using the IC20 protocol on the Cosmos Hub. If this proposal passes, there will be IBC assets available in the Bank module of the Hub and ATOM will be available on Zones connected over IBC.
+Iqlusion believes that the IBC software is suffciently stable for small amounts of value transfer. We expect there to be issues with stuck funds and UX confusion but overcoming these issues will only happen once IBC is live.
```

Comment:
```suggestion
Iqlusion believes that the IBC software is sufficiently stable for small amounts of value transfer. We expect there to be issues with stuck funds and UX confusion but overcoming these issues will only happen once IBC is live.
```

Reply:
What is sufficiently small? That isn't enforced at the protocol level right? Just something we need to explain to users, correct?

Reply:
Also, what are stuck funds and how does that happen? What is the severity of that?

Reply:
> Also, what are stuck funds and how does that happen? What is the severity of that?

Stuck funds are funds that get sent over IBC and for some reason the associated IBC client becomes non-updateable. It may expire (2-3 weeks of no update (trusting period)) or become frozen due to misbehaviour (unlikely and comes with more severe consequences)

It probably will happen by users or services creating a client/connection/channel to test sending funds and then forgetting about it for a month or so. The tokens will likely be stuck on the chain it was sent to. I send atoms to chainB and my token is now ibc/{hash} and I cannot redeem it back for atoms

If you send atoms over well-maintained clients (some entity has promised to make sure it doesn't expire) then this shouldn't be an issue

Reply:
there's some other edge cases where funds may become stuck in the process of transferring from one chain to another. But you'd have to initiate the transfer, never relay the packet, and then never update the client

Governance could always decide to vote to update frozen/expired clients, but this should be saved for clients who have a lot of value built upon them